snap/snapcraft.yaml

```diff
@@ -1,42 +1,44 @@
 name: shotwell
-version: '0.32.1'
-summary: Open source photo manager for GNOME
-description: |
-  Shotwell is an open source photo manager for GNOME. It allows you to import,
-  organize, edit, and share your photos.
-
+adopt-info: shotwell
 grade: stable
 confinement: strict
-
 base: core22
-adopt-info: shotwell
+
+slots:
+  # for GtkApplication registration
+  drawing:
```

Comment:
Could you leave a comment explaining this just for my education? :)

Reply:
Whoops, the drawing bit was because i copied that from another yaml and missed fixes that line.  The slot definition tells snapd it needs to provide that dbus name on the session bus.